content/articles/creating-a-simple-chat-app-in-react-using-the-chat-engine-api/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Creating a Simple Chat App in React.js using the Chat Engine API
+---
+
+
+Ever thought of coming up with a chat application, but going the Node.js and or the socket.io way seemed pretty cumbersome for someone solely equipped with front-end skills like React.js? 
+Well, this article fits you right. 
+The reader is brought to an understanding of the Chat engine API that we will be using, to come up with a precise yet functional Direct-Messaging application built in React.js. 
+Approximating to other APIs that deliver to the system, responses from users, and sends back responses of the system to the user, the Chat engine API does this also only with better functionalities.
+
+<!--more-->
+This tool saves you a lot of implementation time you would otherwise need on many lines of code.
+Chat Engine API can implement; 
+ - A group chat application.
+ - Direct messages chat application that would work like Facebook messenger .
+
+### Prerequisites
+You will need to be equipped with what follows next before jumping right to it.
+1. Primary React.js library mastery. Head to [the official documentation](https://reactjs.org/tutorial/tutorial.html) if there's a need to familiarize yourself with React.js.
+2. Installed on your system Node.js, which you could access from [here](https://nodejs.org/en/download/). 
+3. [Node Package Manager (NPM)](https://docs.npmjs.com/cli/v6/commands/npm) know-how.
+
+### What Chat Engine API is?
+The above introduction gave us an overview of this API and we realized how the API could be consumed to build easily; chat services.
+It also allows the hosting of chats through the Rest API it lays out. The chat User Interface(UI), is created from the NPM components this tool comes with. See how the hustle is waved away for us already?
+Also, thanks to its server(s), the chat can be hosted from here(the server).
+
+
+### Creating the Application
+This minute, let's begin.
+- Head to your windows terminal and navigate to a new directory named to your preference, and create a  new react app. You could opt to use the Vite tool instead whose step-by-step guide is taught in [this](https://www.section.io/engineering-education/creating-a-react-app-using-vite/) article. 
+  Else, run the command below with the project name replaced with an application name of your choice. You may opt to use ***yarn create-react-app app-name***. 
```

Comment:
Simplify your sentences, and do not let users guess of what should happen, give detailed description. For example, say ` On your terminal, cd to project directory and run the following commands`. be precise.